### PR TITLE
(style) Will properly navigate to testimonails on nav

### PR DESCRIPTION
**Pull request by dshelters**
Also renders testimonials dynamically from a list. Does not fix issue where someone tried to navigate to a testimony or feature while not on the homepage.

### CODE REVIEW

File: client/components/homepage/testimonials/Testimonials.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    nextArrow: false,
+    prevArrow: false,
+    autoplaySpeed: 2000
+  };
+
+  return (
+    <section id="testimonials" className="section testimonials background-gray-lightest">
+      <div className="container">
+        <h2 className="text-center">
+          What the world is saying about Ribbit
```
Comment (misteryeo) [inline client/components/homepage/testimonials/Testimonials.jsx]: Personal preference, feels like this looks a bit long and drawn out. Would prefer shorter wording.

File: client/components/homepage/contact/ContactItem.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const ContactItem = (props) => (  
+const ContactItem = props => (
   <div className="col-xs-12 col-sm-6 col-md-3 text-center">
     <div className="box-simple">
       <img src={`../../img/${props.name}.jpg`} className="icon" />
```
Comment (misteryeo) [inline client/components/homepage/contact/ContactItem.jsx]: Are we able to make these images larger and more prominent? Not as icons?

Comment (dshelters) [inline client/components/homepage/contact/ContactItem.jsx]: I can go back and enlarge the images to make them more prominent. Didn't know that was on the agenda of things to do 😄
